dashboard/src/t5gweb/libtelco5g.py

```diff
@@ -252,12 +252,31 @@ def create_cards(cfg, new_cases, action="none"):
         else:
             novel_cases.append(case)
         assignee = None
+        previous_issues_query = (
```

Comment:
If you could, I think it would be a good idea to put as much of this logic into a separate function - `create_cards()` is getting kind of long.